chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -272,8 +272,10 @@
 <!ENTITY zotero.feedSettings.title.label                "Title:">
 <!ENTITY zotero.feedSettings.refresh.label1             "Update feed every">
 <!ENTITY zotero.feedSettings.refresh.label2             "hour(s)">
-<!ENTITY zotero.feedSettings.cleanupAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanupAfter.label1         "Remove articles after ">
 <!ENTITY zotero.feedSettings.cleanupAfter.label2         "day(s)">
+<!ENTITY zotero.feedSettings.cleanupReadAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanupReadAfter.label2         "day(s)">
```

Comment:
I am kinda unsure about this wording in general. "Remove read feed items after" and "Remove feed items after" is kinda confusing, as the distinction is one word. It could be "Remove read feed items after" and "Do not retain any feed items older than", but I am not sure that clears it up either.